homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js

```diff
@@ -0,0 +1,13 @@
+function func(x) {
+    if (typeof x !== 'number') {
+        return 'Not a number';
+    }
+    return (Math.log(x) / Math.log(2)) % 1 === 0;
```

Comment:
the condition of task is not to use `Math` methods

Reply:
> the condition of task is not to use `Math` methods

fixed